core/src/main/scala/kafka/server/BrokerServer.scala

```diff
@@ -696,6 +721,10 @@ class BrokerServer(
         CoreUtils.swallow(socketServer.shutdown(), this)
       if (brokerTopicStats != null)
         CoreUtils.swallow(brokerTopicStats.close(), this)
+      if (sharePartitionManager != null)
+        CoreUtils.swallow(sharePartitionManager.close(), this)
+      if (persister != null)
+        CoreUtils.swallow(persister.stop(), this)
```

Comment:
This is unnecessary since `sharePartitionManager.close()` already calls `persister.stop()`.

Reply:
Ohh yes, you are right, missed it. Thanks. I have pushed the change.